website/addons/zotero/tests/test_models.py

```diff
@@ -34,337 +42,20 @@ def test_handle_callback(self):
-            )
-        )
-
-    def test_set_auth_wrong_user(self):
-        external_account = ZoteroAccountFactory()
-        self.user.external_accounts.append(external_account)
-        self.user.save()
-
-        with assert_raises(PermissionsError):
-            self.node_settings.set_auth(
-                external_account=external_account,
-                user=UserFactory()
-            )
-
-    def test_deauthorize(self):
-        self.node_settings.external_account = ZoteroAccountFactory()
-        self.node_settings.zotero_list_id = 'something'
-        self.node_settings.user_settings = self.user_settings
-        self.node_settings.save()
-
-        assert_true(self.node_settings.zotero_list_id)
-        self.node_settings.deauthorize(auth=Auth(self.user))
-        self.node_settings.save()
-        assert_is(self.node_settings.user_settings, None)
-        assert_is(self.node_settings.zotero_list_id, None)
-
-        last_log = self.node.logs[-1]
-        assert_equal(last_log.action, 'zotero_node_deauthorized')
-        params = last_log.params
-        assert_in('node', params)
-        assert_in('project', params)
-
-    def test_clear_auth(self):
-        self.node_settings.external_account = ZoteroAccountFactory()
-        self.node_settings.zotero_list_id = 'something'
-        self.node_settings.user_settings = self.user_settings
-        self.node_settings.save()
-
-        self.node_settings.clear_auth()
-
-        assert_is_none(self.node_settings.external_account)
-        assert_is_none(self.node_settings.zotero_list_id)
-        assert_is_none(self.node_settings.user_settings)
-
-    def test_set_target_folder(self):
-        folder_id = 'fake-folder-id'
-        folder_name = 'fake-folder-name'
-
-        external_account = ZoteroAccountFactory()
-        self.user.external_accounts.append(external_account)
-        self.user.save()
-
-        self.node_settings.set_auth(
-            external_account=external_account,
-            user=self.user
-        )
-
-        assert_is_none(self.node_settings.zotero_list_id)
-
-        self.node_settings.set_target_folder(
-            folder_id,
-            folder_name,
-            auth=Auth(user=self.user),
-        )
-
-        # instance was updated
-        assert_equal(
-            self.node_settings.zotero_list_id,
-            'fake-folder-id',
-        )
-
-        # user_settings was updated
-        # TODO: the call to grant_oauth_access should be mocked
-        assert_true(
-            self.user_settings.verify_oauth_access(
-                node=self.node,
-                external_account=external_account,
-                metadata={'folder': 'fake-folder-id'}
-            )
-        )
-
-        log = self.node.logs[-1]
-        assert_equal(log.action, 'zotero_folder_selected')
-        assert_equal(log.params['folder_id'], folder_id)
-        assert_equal(log.params['folder_name'], folder_name)
-
-    def test_has_auth_false(self):
```

Comment:
Is this test implemented in the testing mixin? I couldn't find one similar. (sorry, this is a dupe of one I added to a commit on accident).